Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -144,7 +143,20 @@ public void periodic( )
   @Override
   public void simulationPeriodic( )
   {
-    // This method will be called once per scheduler run when in simulation
+    // if (m_allowPoseEstimate)
```

Comment:
Not sure why this was added to Compe2023-imported either. If not needed, just revert this change.